test/print_test_stats.py

```diff
@@ -135,9 +135,22 @@ def send_report_to_scribe(reports):
 def send_report_to_s3(reports, *, total_seconds):
     job = os.environ.get('CIRCLE_JOB')
     sha1 = os.environ.get('CIRCLE_SHA1')
+    branch = os.environ.get('CIRCLE_BRANCH', '')
```

Comment:
Out of curiosity, is there a reason this uses the two-arg version of `os.environ.get` whereas the preceding two lines use the one-arg version?

Reply:
I've encounter some situation where `CIRCLE_BRANCH` was unset. but `CIRCLE_JOB/SHA1` was always set.

Reply:
Hmm but if you look at [the other `CIRCLE_*` env vars used in this script](https://github.com/walterddr/pytorch/blob/f23e771e9ba2384376e53c3741fb08a037f5870f/test/print_test_stats.py#L83-L88), they're also accessed with the one-arg version (including `CIRCLE_BRANCH`); should they be changed to use the two-arg version with default `''` as well?

Reply:
good point. we should follow up based on https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables